features/admin.claims.v1/components/edit/local-claim/edit-mapped-attributes-local-claims.tsx

```diff
@@ -162,39 +220,83 @@ export const EditMappedAttributesLocalClaims: FunctionComponent<EditMappedAttrib
+                                                                    `${componentId}-form-attribute-name-input-
+                                                                    ${store.name}` }
+                                                                readOnly={ isReadOnly }
+                                                            />
+                                                        </Grid.Column>
+                                                    </Grid.Row>
+                                                    { ClaimManagementConstants.USER_STORE_CONFIG_SUPPORTED_CLAIMS
+                                                        .includes(claim.claimURI) && (
+                                                        <Grid.Row columns={ 2 } key={ index } verticalAlign="middle">
+                                                            <Grid.Column width={ 6 }>
+                                                                <p>{ t("claims:local.mappedAttributes." +
+                                                                    "enableForUserStore") }</p>
+                                                            </Grid.Column>
+                                                            <Grid.Column width={ 6 }>
+                                                                <Checkbox
+                                                                    checked={ !excludedUserStores.includes(store.name) }
+                                                                    onChange={
+                                                                        (e: React.ChangeEvent<HTMLInputElement>) => {
+                                                                            if (e.target.checked) {
+                                                                                setExcludedUserStores(
+                                                                                    excludedUserStores.filter(
+                                                                                        (name: string) =>
+                                                                                            name !== store.name)
+                                                                                );
+                                                                            } else {
+                                                                                setExcludedUserStores([
+                                                                                    ...excludedUserStores,
+                                                                                    store.name
+                                                                                ]);
+                                                                            }
+                                                                        } }
```

Comment:
Shall we consider separating this out to a function?

Reply:
Moved